### PR TITLE
Fix Mvc dynamic Form DateTimePicker Error

### LINKED ISSUE #16494: Mvc dynamicForm DateTime date picker unworking
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

image

dynamicForm datetime property unworking in modal page

### Reproduction Steps

ViewModel.cs
```
public class CreateEditCategoryViewModel
{
    [Display(Name = "CategoryName")]
    public string Name { get; set; }

    public DateTime BookingTime { get; set; }
}
```
CreateModal.cshtml
```
<abp-dynamic-form abp-model="ViewModel" data-ajaxForm="true" asp-page="CreateModal">
    <abp-modal>
        <abp-modal-header title="@L["CreateCategory"].Value"></abp-modal-header>
        <abp-modal-body>
            <abp-form-content />
        </abp-modal-body>
        <abp-modal-footer buttons="@(AbpModalButtons.Cancel|AbpModalButtons.Save)"></abp-modal-footer>
    </abp-modal>
</abp-dynamic-form>
```

### Expected behavior

Click the BookingTime input box to pop up the date picker

### Actual behavior

Nothing happens

### Regression?

7.1.1 normal

### Known Workarounds

_No response_

### Version

7.2.1

### User Interface

MVC

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

### Other information

_No response_

### PR DESCRIPTION
**Pull request by salihozkara**
Resolves #16494 

image

### Checklist

- [x] I fully tested it as developer

### How to test it?

Create a modal and have abp-date-picker in the modal. The datepicker should work when you open the modal

### CODE REVIEW

File: framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Form/DatePicker/AbpDatePickerBaseTagHelperService.cs [hunk 1]
```diff
@@ -84,6 +84,11 @@ public async override Task ProcessAsync(TagHelperContext context, TagHelperOutpu
...
         AddPlaceholderAttribute(TagHelperOutput);
         AddInfoTextId(TagHelperOutput);
+        var optionsAttribute = GetAttributeAndModelExpression<DatePickerOptionsAttribute>(out var modelExpression);
+        if (optionsAttribute != null)
+        {
+            TagHelper.SetDatePickerOptions(optionsAttribute.GetDatePickerOptions(modelExpression.ModelExplorer));
+        }
 
         // Open and close button
         var openButtonContent = TagHelper.OpenButton
```
Comment (blackWins) [inline framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Form/DatePicker/AbpDatePickerBaseTagHelperService.cs:94]: The `DisabledInput` etc attribute can be detected here

Comment (blackWins) [reply framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Form/DatePicker/AbpDatePickerBaseTagHelperService.cs:94]:
```
var openButtonContent = TagHelper.OpenButton && !DisabledInput 
            ? await ProcessButtonAndGetContentAsync(context, output, "calendar", "open")
```

**Comment by blackWins** [comment]
AbpDatePickerBaseTagHelperService.cs line: 331
Whether it is better to set the `AutoUpdateInput` default to `true`
```
 if (options.AutoUpdateInput != null)
        {
            attrList.Add("data-auto-update-input", options.AutoUpdateInput.ToString().ToLowerInvariant());
        }
```

**Comment by salihozkara** [comment]
> AbpDatePickerBaseTagHelperService.cs line: 331 Whether it is better to set the `AutoUpdateInput` default to `true`
> 
> ```
>  if (options.AutoUpdateInput != null)
>         {
>             attrList.Add("data-auto-update-input", options.AutoUpdateInput.ToString().ToLowerInvariant());
>         }
> ```

Does it work with AutoUpdateInputAttribute?

**Comment by blackWins** [comment]
> > AbpDatePickerBaseTagHelperService.cs line: 331 Whether it is better to set the `AutoUpdateInput` default to `true`
> > ```
> >  if (options.AutoUpdateInput != null)
> >         {
> >             attrList.Add("data-auto-update-input", options.AutoUpdateInput.ToString().ToLowerInvariant());
> >         }
> > ```
> 
> Does it work with AutoUpdateInputAttribute?

Yes，when i set `AutoUpdateInput = true` the value is show

image

**Comment by salihozkara** [comment]
@blackWins 
I made a wrong sentence, what do you think if I create an attribute instead of making it true by default?

**Comment by weiwxg** [comment]
When will this bug fix be released?

**Comment by salihozkara** [comment]
Transfered -> https://github.com/abpframework/abp/pull/16561

**Pull request closed without merging**